config/database.yml

```diff
@@ -1,25 +1,20 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
+  username: postgres
+  password: xxxxx #<-- which you have defiend while installing postgresql
```

Comment:
You shouldn't be putting passwords onto soruce control. If you need to set your password, use an env variable, like in the production config.